client/layout/guided-tours/theme-sheet-welcome-tour.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { overEvery as and, negate as not } from 'lodash';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	Continue,
+	Next,
+	Quit,
+	Step,
+	Tour,
+	makeTour,
+} from 'layout/guided-tours/config-elements';
+import {
+	isAbTestInVariant,
+	isEnabled,
+	isNewUser,
+	previewIsShowing,
+} from 'state/ui/guided-tours/contexts';
+import { isDesktop, isMobile } from 'lib/viewport';
+
+const PickActivateStep = () => (
+	<p>
+		{ translate(
+			'This will activate the design you’re currently seeing on your site.'
```

Comment:
I wasn't aware we're using `’`. Shouldn't we use `'` everywhere for consistency? And then use double quotes here for quoting the string?